appengine/flexible/endpoints/README.md

```diff
@@ -40,7 +40,7 @@ Open the `swagger.yaml` file and in the `host` property, replace
 
 Then, deploy the sample using `gcloud`:
 
-    gcloud preview app deploy app.yaml
+    gcloud app deploy app.yaml
```

Comment:
technically don't need to call out the app.yaml (`gcloud app deploy` by itself is so pretty)